csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3307,7 +3324,46 @@ Example which fails the test:
 
 Informatiove tests provide insights in common mistakes and bad practices. They MAY fail at a valid CSAF document. It is up to the issuing party to decide whether this was an intended behavior and can be ignore or should be treated. These tests may include information about recommended usage. A program MUST handle a test failure as a information.
 
-# 5 Safety, Security, and Data Protection Considerations
+# 5 Distributing CSAF documents
+
+This section lists requirements and roles defined for distributing CSAF documents. The first subsection provides all requirements - the second one the roles. It is mandatory to fulfill the basic role "CSAF publisher".
+
+## 5.1 Requirements
+
+The requirements in this subsection are consecutively numbered to be able to refer to them directly. The order does not give any hint about the importance.
+
+### 5.1.1 Requirement 1: Valid CSAF document
+
+The document is a valid CSAF document (cf. Conformance clause 1).
+
+### 5.1.2 Requirement 2: Filename
+
+The CSAF document has a filename according to the rules in section 3 (cf. section 3.2.1.12.4).
+
+### 5.1.3 Requirement 3: TLS
+
+The CSAF document is retrievable from a website which uses TLS for encryption and server authenticity. The CSAF document MUST not be downloadable from a location which does not encrypt the transport.
```

Comment:
I do not understand why we put emphasis on encryption here. In many cases the document will be public, so we should focus on server authentication and data integrity. 
Do we want to also force TLS? What about PGP to ensure those requirements? I am thinking about the typical package mirrors of Linux distributions that are still available via HTTP and FTP.